website/docs/cloud-docs/api-docs/policy-sets.mdx

```diff
@@ -797,6 +829,53 @@ curl \
   https://app.terraform.io/api/v2/policy-sets/polset-3yVQZvHzf5j3WRJ1/relationships/workspaces
 ```
 
+## Attach a Policy Set to exclusions
+
+`POST /policy-sets/:id/relationships/excludedWorkspaces`
```

Comment:
Is the endpoint called `excludedWorkspaces` or `excluded`?

Reply:
it's exclusions now, fixed!